openshift/charts/cluster-api-operator/templates/ocp-capa-provider.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: v1
```

Comment:
please remove this file from this PR , will include it in another PR with jira ACM-14308